internal/commands/scan.go

```diff
@@ -2216,3 +2220,27 @@ func deprecatedFlagValue(cmd *cobra.Command, deprecatedFlagKey, inUseFlagKey str
 	}
 	return flagValue
 }
+
+func validateCreateScanFlags(cmd *cobra.Command) error {
+	branch := viper.GetString(commonParams.BranchKey)
+	if branch == "" {
+		return errors.Errorf("%s: Please provide a branch", failedCreating)
+	}
+	exploitablePath, _ := cmd.Flags().GetString(commonParams.ExploitablePathFlag)
+	lastSastScanTime, _ := cmd.Flags().GetString(commonParams.LastSastScanTime)
+	if !strings.Contains(strings.ToLower(actualScanTypes), commonParams.SastType) &&
+		(exploitablePath != "" || lastSastScanTime != "") {
+		return errors.Errorf("Please to use either --exploitable-path or --last-sast-scan-time flags in SCA, " +
```

Comment:
I would remove this concatenation, and do a single string and remove the "please" (I would check the messages with a PM)